extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -437,17 +437,26 @@ export class SubscriptionService {
    * @return {!Promise<boolean>}
    */
   delegateActionToLocal(action) {
-    const localPlatform = /** @type {LocalSubscriptionPlatform} */ (
-      dev().assert(this.platformStore_.getLocalPlatform(),
-          'Local platform is not registered'));
-    // TODO: add which service is passing this event
+    this.delegateActionToService(action, 'local');
```

Comment:
`return` missing? It's declared as `@return {!Promise...}`. That, I'd expect, should be "noticed" by a test?

Reply:
Done